component/monitoring.libsonnet

```diff
@@ -0,0 +1,103 @@
+local cm = import 'lib/cert-manager.libsonnet';
+local com = import 'lib/commodore.libjsonnet';
+local kap = import 'lib/kapitan.libjsonnet';
+local kube = import 'lib/kube.libjsonnet';
+local prom = import 'lib/prometheus.libsonnet';
+local inv = kap.inventory();
+// The hiera parameters for the component
+local params = inv.parameters.openshift4_api;
+
+local nsName = 'syn-monitoring-openshift4-api';
+
+local promInstance =
+  if params.monitoring.instance != '' then
+    params.monitoring.instance
+  else
+    inv.parameters.prometheus.defaultInstance;
+
+local apiServerMonitor = function(name, selector)
+  prom.ServiceMonitor(name) {
+    metadata+: {
+      namespace: nsName,
+    },
+    spec+: {
+      endpoints: [
+        {
+          bearerTokenFile: '/var/run/secrets/kubernetes.io/serviceaccount/token',
```

Comment:
Do you really need both the mTLS certificates and a bearer token?

Reply:
I did not test without. The RedHat ones have both.